source/transfer_list.rst

```diff
@@ -657,5 +658,86 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _tf_entries:
+
+Trusted Firmware related entries
```

Comment:
Trusted Firmware-related entries

or

Entries related to Trusted Firmware

Reply:
Fixed.